src/Homework4/07_nod_evklid.py

```diff
@@ -0,0 +1,13 @@
+"""
+Оглянемся назад
+Даны два натуральных числа. Вычислите их наибольший общий делитель
+при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+small, big = 215484565498465686464846546846516841646568166851, 462
+while big and small:
+    if big > small:
+        big %= small
+    else:
+        small %= big
+print(big + small)
```

Comment:
it's a cheat. algorithm doesn't expect result in sum :) 
you can use ternary operator.

Reply:
Euclid would understand :-)